Initial German translation commit

flaymeariver opened the pull request:
Added files with the initial German translations of the given variables

flaymeariver committed:
German Translation

Added files with German translations of the given variables

NoelDavies commented on de-DE/Common.js:
```diff
@@ -0,0 +1,13 @@
+    'Web': 'Web Sources',
+    'Nuzlocke': 'Nuzlocke Tools',
+    'Settings': 'Einstellungen',
+    'About': 'About'
```
Are the following correct German translations?

- Sessions
- Web Sources
- "Tools"
- About
-

NoelDavies commented on de-DE/Common.js:
```diff
@@ -0,0 +1,13 @@
+export default {
+  'Navigation': {
```
Are the following correct German translations?

- Sessions
- Web Sources
- "Tools"
- About
-

flaymeariver replied:
These words are regularly used as so-called anglicisms in the German language in this context. There are existing German translations, but those would sound strange even to native speakers as myself.

NoelDavies commented on de-DE/Settings.js:
```diff
@@ -0,0 +1,47 @@
+  },
+  'Fields': {
+    'SessionName': {
+      'Titles': 'Session-Name',
```
Is this really hyphenated in German?

flaymeariver replied:
This is quite a deep topic in the German language, short answer: yes

Long answer: It depends on how accepted an anglicism is to the German language. Generally it is regarded to just rather hypenate the word connection, if you are unsure. In this particular case, session is a regularly used term in IT, but it isn't accepted enough into the German language to regard it as a native word. This distinction is a bit harder in the word Team-Management where even German dictionaries seem to argue about the correct way to spell it. Therefore I went for the safer approach to hyphenate it. If you're interested in the theory, this article explains it kind of nice: https://www.scribbr.de/wissenschaftliches-schreiben/zusammengesetzte-anglizismen/ (would need Google Translate I suppose)

NoelDavies replied:
Dude as long as you're happy with it:P

flaymeariver replied:
haha sorry, just my notorious curiousity 😄

NoelDavies commented on de-DE/index.js:
```diff
@@ -0,0 +1,16 @@
+
+export default {
+  label: 'Deutsch (DE)',
+  code: 'de-DE',
```
This should just be "de"

flaymeariver replied:
done

NoelDavies reviewed (commented):
Can you rename this folder to just `de`?

flaymeariver committed:
Renamed de-DE folder to de

Renamed the folder containing the German translation from de-DE to de upon request.

flaymeariver committed:
Revert "Renamed de-DE folder to de"

This reverts commit ea8e470cc0dc20516c4c588b9ca61e6cb69238a1.

flaymeariver committed:
Initial German translation commit

Initial German translation commit.
-Changed folder name from de-DE to de upon request
-Changed language code in index.js from de-DE to de
